test/test/CompilerTest.scala

```diff
@@ -465,7 +492,7 @@ abstract class CompilerTest {
 
   /** Reads the existing files for the given test source if any. */
   private def getExisting(dest: Path): ExistingFiles = {
-    val content: Option[Option[String]] = processFileDir(dest, f => f.safeSlurp, d => Some(""))
+    val content: Option[Option[String]] = processFileDir(dest, f => try Some(f.slurp("UTF8")) catch {case io: java.io.IOException => Some(io.toString())}, d => Some(""))
```

Comment:
`safeSlurp` is problematic in that it can't take an encoding as an argument, and uses the encoding when the `File` is created, and many ways to create a `File` don't have the opportunity to pass an encoding either (in this case `ifFile`).

That said, it can still go from this line.